README.md

```diff
@@ -0,0 +1,91 @@
+So, when this template is run, user will be required to accept the <a href="https://www.oracle.com/downloads/licenses/standard-license.html">OTN Free Developer License Agreement</a> and also provide OTN credentials (username and password), to download the Oracle Weblogic Install Kit and Oracle JDK.
+
+
+<h3>Using the template</h3>
+
+<h4>Perform string substitution to generate the necessary artifacts for deployment or uploading to the Azure Cloud Partner Portal</h4>
+
+* Install Apache Maven.  This project uses Apache Maven to do simple
+  string substitution for several required parameters in the templates.
+  
+* From the top level run `mvn clean install`.
+
+* The templates end up in `arm-oraclelinux-wls/arm-oraclelinux-wls/target/arm`.  Change to that directory to run the templates.
+
+<h4>Once you have performed the string substitution, you can deploy the template via the command line</h4>
+
+**PowerShell** 
+
+*#use this command when you need to create a new resource group for your deployment*
+
+*New-AzResourceGroup -Name &lt;resource-group-name&gt; -Location &lt;resource-group-location&gt; 
+
+*New-AzResourceGroupDeployment -ResourceGroupName &lt;resource-group-name&gt; -TemplateFile mainTemplate.json*
+
+**Command line**
+
+```
+az group create --name &lt;resource-group-name&gt; --location &lt;resource-group-location&gt;
+
+az group deployment create --resource-group &lt;resource-group-name&gt; --template-file mainTemplate.json  --parameters @parametersFile.json
+```
+
+For example:
+
+```
+az group deployment create --resource-group 20191001-01-my-rg --parameters @my-parameters.json --template-file arm-oraclelinux-wls/target/arm/mainTemplate.json
+```
+
+If you are new to Azure virtual machines, see:
+
+- [Azure Virtual Machines](https://azure.microsoft.com/services/virtual-machines/).
+- [Azure Linux Virtual Machines documentation](https://docs.microsoft.com/azure/virtual-machines/linux/)
+- [Azure Windows Virtual Machines documentation](https://docs.microsoft.com/azure/virtual-machines/windows/)
+- [Template reference](https://docs.microsoft.com/azure/templates/microsoft.compute/allversions)
+- [Quickstart templates](https://azure.microsoft.com/resources/templates/?resourceType=Microsoft.Compute&pageNumber=1&sort=Popular)
+
+If you are new to template deployment, see:
+
+[Azure Resource Manager documentation](https://docs.microsoft.com/azure/azure-resource-manager/)
+
+## Considerations for CI/CD
+
+<h3>Running the tests</h3>
+
+Microsoft provides template validation tests in the Git repo for [azure-quickstart-templates](https://github.com/Azure/azure-quickstart-templates/tree/master/test/template-tests).  This project has maven configuration to run those tests against the ARM template.  This is useful when building the template as part of a CI/CD pipeline.
```

Comment:
This project has maven configuration -> This project includes a Maven configuration